src/logging.h

```diff
@@ -67,6 +69,8 @@ namespace BCLog {
         FILE* m_fileout GUARDED_BY(m_cs) = nullptr;
         std::list<std::string> m_msgs_before_open GUARDED_BY(m_cs);
         bool m_buffering GUARDED_BY(m_cs) = true; //!< Buffer messages before logging can be started.
+        std::map<std::string, uint64_t> m_bytes_logged_per_source_location GUARDED_BY(m_cs);
```

Comment:
Maybe `unordered_map`? we have no ordering constraing here.

Reply:
Fixed!